src/main/java/at/hannibal2/skyhanni/features/rift/area/stillgorechateau/RiftBloodEffigies.kt

```diff
@@ -74,7 +71,7 @@ class RiftBloodEffigies {
         if (!isEnabled()) return
 
         val line = event.newList.firstOrNull { it.startsWith("Effigies:") } ?: return
-        val hearts = effegieHeartPattern.matchMatcher(line) {
+        val hearts = effigieTimerPattern.matchMatcher(line) {
             group("hearts")
         } ?: return
```

Comment:
How can this work. There is no group called "hearts in effigieTimerPattern but there is in heartsPattern. So it should be the heartsPattern not effigieTimerPattern. Or is something else wrong?